lib/verbalize/action.rb

```diff
@@ -17,20 +17,24 @@ def self.included(target)
 
     module ClassMethods
       def input(*required_keywords, optional: [])
-        @inputs = required_keywords
+        @required_inputs = required_keywords
         @optional_inputs = optional
 
         class_eval Build.call(required_keywords, Array(optional))
       end
 
-      def inputs
-        @inputs || []
+      def required_inputs
+        @required_inputs || []
       end
 
       def optional_inputs
         @optional_inputs || []
       end
 
+      def inputs
+        required_inputs + optional_inputs
```

Comment:
I think this is fine the way it is. 

I do agree it would be better if we had a clear use case, but I think this is so simple it's not going to be a problem.